packages/next-app/src/components/Sidebar.tsx

```diff
@@ -0,0 +1,85 @@
+import { MagnifyingGlassIcon } from "@radix-ui/react-icons";
+import Link from "next/link";
+import { useState } from "react";
```

Comment:
I have a feeling that if this is to live under `/components` its API must not be tied to your use case in daoadmin, otherwise it's just gonna be impossible to assume that these components can be used by any app and/or whomever wants to reuse a component will have to spend a lot of time refactoring it and making sure it doesn't break anything in the original implementation. WDYT?

Reply:
Now looking I was thinking the same, for now I'll move to `/daoadmin` and think on how to implement as a component for all apps later on

Reply:
defer to you, but I don't see us being in a hurry such that taking more time here would lead to a tradeoff in delaying our delivery time since we don't have a hard delivery timeline, so would really optimize for learning and code quality until we do (and also because, when we do, these shortcuts will _also_ cost us time to deliver new things and adapt what we've done)